src/components/tips/tip-teaser.tsx

```diff
@@ -2,11 +2,23 @@ import {useRouter} from 'next/navigation'
 import Image from 'next/image'
 import Link from 'next/link'
 import Balancer from 'react-wrap-balancer'
+import {
+  CheckCircleIcon as CheckCircleIconOutline,
+  ArrowsExpandIcon,
+} from '@heroicons/react/outline'
+import {CheckCircleIcon, CheckIcon} from '@heroicons/react/solid'
+import {trpc} from 'app/_trpc/client'
 
 export const TipTeaser: React.FC<{tip: any}> = ({tip}) => {
   const {title, muxPlaybackId} = tip
   const thumbnail = `https://image.mux.com/${muxPlaybackId}/thumbnail.png?width=720&height=405&fit_mode=preserve`
   const router = useRouter()
+  const data = tip?.eggheadRailsLessonId
```

Comment:
I wonder if instead of data you could just have a boolean on each of these ternary results. Then you wouldn't need the second tipCompleted constant

```
? trpc.tips.loadTipProgress.useQuery({
        id: tip?.eggheadRailsLessonId,
      }).data?.tipCompleted
    : false
```